### ISSUE TITLE
Add repo description + README

### ISSUE THREAD

**Issue by lachlanjc** [hackclub/hack-coins#5]
Would be helpful so other people know what this project is :)

**Comment by EshanDoes** [comment]
Yeah, it would be helpful if there was a repo description. Hopefully there can be one soon.
